components/device-mgt/org.wso2.carbon.device.mgt.api/src/main/java/org/wso2/carbon/device/mgt/jaxrs/api/impl/UserImpl.java

```diff
@@ -195,31 +194,23 @@ public Response updateUser(UserWrapper userWrapper, @QueryParam("username") Stri
                             new String(decodedBytes, "UTF-8"));
```

Comment:
Ack, Fixed in commit https://github.com/wso2/carbon-device-mgt/pull/303/commits/051493db6a3fdb2e75cc7d1f48cb3ed407622312